src/sidebar/components/MarkdownView.tsx

```diff
@@ -35,21 +58,80 @@ export default function MarkdownView({
     });
   }, [markdown]);
 
+  useEffect(() => {
+    if (!hasMentions) {
+      return () => {};
+    }
+
+    const listenerCollection = new ListenerCollection();
+    const foundMentions = renderMentionTags(content.current!, mentions);
+
+    listenerCollection.add(
```

Comment:
Was there a reason not to use `onMouseEnterCapture` and `onMouseLeaveCapture` on the `content` div? (`onMouseOver` and `onMouseOut` could work as well with some other modifications I think).

Reply:
The main reason is that `renderMentionTags` needs to be called in a side effect, so we would have to track the result in a ref.

Defining the event handlers here simplified that is it no longer needs to be referenced outside. But we can explore the other option.